src/isar/config/settings.py

```diff
@@ -210,3 +210,28 @@ class Config:
 
 
 settings = Settings()
+
+
+def default_capabilities():
+    return ["drive_to_pose", "take_image"]
+
+
+class RobotSettings(BaseSettings):
+    def __init__(self) -> None:
+        try:
+            with pkg_resources.path(
+                f"{settings.ROBOT_PACKAGE}.config", f"settings.env"
+            ) as path:
+                env_file_path = path
+        except ModuleNotFoundError:
+            env_file_path = None
+        super().__init__(_env_file=env_file_path)
+
+    CAPABILITIES: List[str] = Field(default_factory=default_capabilities)
+
+    class Config:
+        env_file_encoding = "utf-8"
+        case_sensitive = True
+
+
+robot_settings = RobotSettings()
```

Comment:
I suggest to make this a 'private' member function to keep it gathered together where it is used.
```suggestion

class RobotSettings(BaseSettings):
    def __init__(self) -> None:
        try:
            with pkg_resources.path(
                f"{settings.ROBOT_PACKAGE}.config", f"settings.env"
            ) as path:
                env_file_path = path
        except ModuleNotFoundError:
            env_file_path = None
        super().__init__(_env_file=env_file_path)

    def _default_capabilities():
        return ["drive_to_pose", "take_image"]

    CAPABILITIES: List[str] = Field(default_factory=_default_capabilities)

    class Config:
        env_file_encoding = "utf-8"
        case_sensitive = True


robot_settings = RobotSettings()
```